seed_data/management/commands/alter_data.py

```diff
@@ -177,55 +173,39 @@ def add_future_run(user=None, course=None, now=None):  # pylint: disable=unused-
     return CourseRun.objects.filter(course=course, edx_course_key__startswith=NEW_COURSE_RUN_PREFIX).delete()
 
 
-def get_past_course_run(user=None, course=None, now=None, add_if_exists=False):
-    """Loop through past course runs and find one without CachedCurrentGrade data"""
-    chosen_past_run = None
+def get_past_ungraded_course_run(user=None, course=None, now=None):
+    """Loop through past course runs and find one without grade data"""
     past_runs = CourseRun.objects.filter(
         course=course,
         end_date__lt=now,
     ).exclude(end_date=None).order_by('-end_date').all()
     for past_run in past_runs:
         if not (CachedCurrentGradeHandler(user).exists(past_run) or
                 CachedCertificateHandler(user).exists(past_run)):
-            chosen_past_run = past_run
-        elif not add_if_exists:
-            raise Exception("Past failed/passed course run already exists (id: {}, title: {})".format(
-                past_run.id,
-                past_run.title
-            ))
-    if not chosen_past_run:
-        raise Exception("Can't find past run w/o CachedCurrentGrade")
-    return chosen_past_run
+            return past_run
+    raise CommandError("Can't find past run that isn't already passed/failed for Course '{}'".format(course.title))
 
 
 @accepts_or_calculates_now
-def add_past_failed_run(user=None, course=None, now=None, grade=DEFAULT_FAILED_GRADE, add_if_exists=False):
+def add_past_failed_run(user=None, course=None, now=None, grade=DEFAULT_FAILED_GRADE):
     """Adds a past failed course run for a given user and course"""
-    chosen_past_run = get_past_course_run(user, course, now, add_if_exists)
-
-    # Add enrollment and failed grade for course run
-    CachedEnrollmentHandler(user).set_or_create(course_run=chosen_past_run)
-    CachedCurrentGradeHandler(user).set_or_create(course_run=chosen_past_run, grade=grade)
-    return chosen_past_run
+    ungraded_past_run = get_past_ungraded_course_run(user, course, now)
+    return set_to_failed(user=user, course_run=ungraded_past_run, grade=grade)
 
 
 @accepts_or_calculates_now
-def add_past_passed_run(user=None, course=None, now=None, grade=DEFAULT_GRADE, add_if_exists=False):
+def add_past_passed_run(user=None, course=None, now=None, grade=DEFAULT_GRADE):
     """Adds a past passed course run for a given user and course"""
-    chosen_past_run = get_past_course_run(user, course, now, add_if_exists)
-    # Add enrollment and  certificate for course run
-    CachedEnrollmentHandler(user).set_or_create(course_run=chosen_past_run)
-    CachedCertificateHandler(user).set_or_create(course_run=chosen_past_run, grade=grade)
-
-    return chosen_past_run
+    ungraded_past_run = get_past_ungraded_course_run(user, course, now)
+    return set_to_passed(user=user, course_run=ungraded_past_run, grade=grade)
 
 
-def _formatted_datetime(dt, dt_format='%m/%d/%Y'):
+def _formatted_datetime(dt, dt_format='%m/%d/%Y %H:%M:%S'):
```

Comment:
i could. i had it in this format for the benefit of jared and any other non-dev folks who might want to see `course_info` output (I figured it would be a little prettier than ISO). since i'm adding time into the output i think it makes sense to go ahead and change it